src/types.rs

```diff
@@ -220,3 +220,40 @@ impl FromStr for SourceLocation {
         }
     }
 }
+
+/// Process Identifier.
+///
+/// # Examples
+///
+/// The default value:
+///
+/// ```
+/// use sloggers::types::ProcessID;
+///
+/// assert_eq!(ProcessID::default(), ProcessID(false));
+/// ```
+#[allow(missing_docs)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub struct ProcessID(pub bool);
```

Comment:
In my opinion, it is excessive to define a new struct for this purpose (simple boolean type is sufficient).

Reply:
I disagree, encapsulation of types prevents weird problems like multiple parameters of same type being confused in sequence and so on. Plus, it looks more consistsent with the other features

Reply:
> encapsulation of types prevents weird problems

Generally, I agree with your opinion. However, wrapping every basic types  feels to be cumbersome (e.g., users are required to import many types to use the library). Besides, in this case of the process ID option, I think a subtle situation such as "multiple parameters of same type being confused in sequence" will hardly occur. 

> it looks more consistsent with the other features

Currently, `sloggers` uses bare types for simple options (e.g., [`rotate_compress`](https://docs.rs/sloggers/0.3.1/sloggers/file/struct.FileLoggerConfig.html#structfield.rotate_compress), [`truncate`](https://docs.rs/sloggers/0.3.1/sloggers/file/struct.FileLoggerConfig.html#structfield.truncate), [`channel_size`](https://docs.rs/sloggers/0.3.1/sloggers/file/struct.FileLoggerConfig.html#structfield.channel_size)).